woocommerce/woocommerce
# Update tag component styling

### All Submissions:






### Changes proposed in this Pull Request:

Update tag styling based on new designs

<img width="343" alt="Screen Shot 2022-08-24 at 2 32 12 PM" src="https://user-images.githubusercontent.com/10561050/186530118-ac9e7652-86c0-4dc3-a80c-343ee5839a7f.png">
<img width="208" alt="Screen Shot 2022-08-24 at 2 32 01 PM" src="https://user-images.githubusercontent.com/10561050/186530122-fb7fdd8f-f750-4c23-a664-1ecb77a4e908.png">


### How to test the changes in this Pull Request:

1. `npm run storybook`
2. Check the `Tag` component and other components using it such as `SelectControl` and `experimental/SelectControl`
3. Back in WooCommerce Admin, go through the analytics reports and select some tags using filters
4. Make sure styling is okay

### Other information:




### FOR PR REVIEWER ONLY:
